### PR TITLE
Regression: Replace placeholders for room content

### PR DESCRIPTION
**Pull request by tiagoevanp**

### CODE REVIEW

File: server/startup/migrations/v228.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { Migrations } from '../../../app/migrations/server';
+import { LivechatRooms, Users } from '../../../app/models/server';
+
+function migrateAgentInformation() {
+	const rooms = LivechatRooms.findAll({
+		// open: true, // should we update only open rooms?
```
Comment (renatobecker) [inline server/startup/migrations/v228.js:6]: Definetely.

File: ee/client/omnichannel/components/CannedResponse/parsePlaceholder.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { IOmnichannelRoom } from '../../../../../definition/IRoom';
+
+export const parsePlaceHolder = (text: string, room: IOmnichannelRoom): string => {
+	const placeholders = text.match(/{{(.*?)}}/g);
+	const placeholderDict = {
+		'contact.name': room.v.name,
+		'contact.email': room.v.visitorEmails.length ? room.v.visitorEmails[0].address : '',
```
Comment (renatobecker) [inline ee/client/omnichannel/components/CannedResponse/parsePlaceholder.ts:7]:
```suggestion
		'contact.email': room.v.visitorEmails?.length ? room.v.visitorEmails[0].address : '',
```

File: ee/client/omnichannel/components/CannedResponse/parsePlaceholder.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+		// 'contact.phone': room.v.status,	//No phone information in livechat form yet
```
Comment (renatobecker) [inline ee/client/omnichannel/components/CannedResponse/parsePlaceholder.ts:8]: This information is stored in the visitor object, not inside room object.

**Comment by KevLehman** [comment]
@MartinSchoeler  @tiagoevanp , can we close this one?

**Comment by MartinSchoeler** [comment]
Closed By https://github.com/RocketChat/Rocket.Chat/pull/22777

**Pull request closed without merging**